app/models/verdict_type.rb

```diff
@@ -4,6 +4,17 @@ class VerdictType < ApplicationRecord
   validates :category, presence: true
   validates :categoryType, presence: true
 
+  CATEGORY_TYPES = { "GUILTY" => ["Found guilty", "Guilty"],
```

Comment:
Where did this list come from? I think there are other verdicts that we will need in order to test Crown Court Outcome Creation. 

The default is to return acquitted if none of the verdicts returned are guilty-type https://github.com/ministryofjustice/laa-court-data-adaptor/blob/a29221014d635bb3f5aa57917a7f20a787f9a402/app/services/crown_court_outcome_creator.rb#L19
So we'd need a NOT_GUILTY (and maybe others if we know verdicts/scenarios here) to test this.